ffc-basic-algorithm/10.js

```diff
@@ -0,0 +1,20 @@
+/*
+Boo who
+Check if a value is classified as a boolean primitive. Return true or false.
+
+Boolean primitives are true and false.
+*/
+
+function booWho(bool) {
+  return bool === true || bool === false;
+}
+
+console.log(booWho(null));
+
+//another way by using typeof
+
+function booWho2(bool) {
+  return typeof bool === "boolean";
```

Comment:
Performance note -> `typeof` and `string` comparisons are slow
Using the triple operator `===` with `typeof` is not required